app/assets/stylesheets/arclight/modules/search_results.scss

```diff
@@ -109,3 +101,7 @@
 .col-no-right-padding {
   padding-right: 0 !important;
 }
+
+.breadcrumb-links a {
+  color: #212529;
```

Comment:
Can we use the Bootstrap variable here `$body-color`?

Reply:
Updated to use variable